src/providers/https.ts

```diff
@@ -262,18 +265,21 @@ export interface CallableContext {
 }
 
 // The allowed interface for an http request for a callable function.
+/** @hidden*/
 interface HttpRequest extends Request {
   body: {
     data: any;
   };
 }
 
+/** @hidden */
 // The format for the http body response to a callable function.
```

Comment:
I know we're hiding this, but just to document it better in the source code, can you revise this to:

The format for an HTTP body response from a Callable function.

Reply:
Done.